great_expectations/expectations/metrics/column_aggregate_metrics/column_quantile_values.py

```diff
@@ -1,10 +1,12 @@
+import ast
 import itertools
 import logging
 import traceback
 from collections.abc import Iterable
 from typing import Any, Dict, List
 
 import numpy as np
+from sqlalchemy import text
```

Comment:
Please remove this import, as you are already using `sa.text` on line 353

Reply:
done